docs/get-started/deployment-and-usage.md

```diff
@@ -285,7 +285,7 @@ labels. The provided template will configure these for you.
 
 NFD-Worker is preferably run as a Kubernetes DaemonSet. This assures
 re-labeling on regular intervals capturing changes in the system configuration
-and mames sure that new nodes are labeled as they are added to the cluster.
+and make sure that new nodes are labeled as they are added to the cluster.
```

Comment:
I think this should be *makes* instead of *make*, right(?)

Reply:
+1

Reply:
Thanks @marquiz , @adrianchiris for suggesting the change, will update this